nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -1698,6 +1698,61 @@ form of the updated JSON.#
 
 An empty subject value or a subject value with an invalid JSON document results in an exception bulletin.
 
+[.function]
+=== jsonPathAdd
+
+*Description*: [.description]#The `jsonPathAdd` function adds a value to an array at the specified JsonPath on
```

Comment:
This should probably say `adds a scalar value`, I don't think it's possible to insert a JSON object right? I can add the word `scalar` during merge if you agree